1-js/01-getting-started/2-manuals-and-specifications/article.md

```diff
@@ -0,0 +1,40 @@
+# Manuais e especificações
+
+Este livro é um *tutorial*. Ele tem como objetivo lhe ajudar a aprender gradualmente a linguagem. Mas uma vez  familiarizado com o básico, irá precisar de outros recursos.
+
```

Comment:
Please,

- '... uma vez  familiarizado ...'
(Although, not quite relevant would you remove the extra space? )